src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,48 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 크기 확인하기")
+    void size() {
+        int size = 3;
+
+        assertThat(numbers.size()).isEqualTo(size);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    @DisplayName("1,2,3 포함하는지 확인하기")
+    void contains(int value) {
+        assertThat(numbers.contains(value)).isTrue();
```

Comment:
```suggestion
        assertThat(numbers).contains(value);
```
(미션 요구사항에서 준 뼈대코드 같지만) 요렇게 쓸 수도 있답니당

Reply:
이런 방법도 있었네용 😳